### PR TITLE
Change body font on Windows

### PR DESCRIPTION
**Pull request by wiredmind**
Calibri is a much more readable and pleasing than Arial.

### CODE REVIEW

File: gui/assets/css/overrides.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 body {
     padding-bottom: 70px;
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "Helvetica Neue", Calibri, Helvetica, Arial, sans-serif;
```
Comment (canton7) [inline gui/assets/css/overrides.css]: Should we have Helvetica before Calibri?

Comment (wiredmind) [inline gui/assets/css/overrides.css]: Windows aliases Helvetica to Arial. If Helvetica is before Calibri in the
font stack Windows will render using Arial.

**Comment by uok** [comment]
I'm not a fan of this suggestion. Calibri displays a wierd "~" and also is smaller than Arial at 16px. Because Bootstrap is a big framework you cannot simply replace a font without adjusting alignments, button-sizes, etc. Please do not merge this and wait for skin/theme system for Syncthing.

**Comment by wiredmind** [comment]
I guess it's a matter of taste and opinion, but I much prefer the look of Calibri over Arial. However I do not agree with your supposition that a typeface change will have breaking consequences on the interface. Bootstrap was build to be customized.

**Comment by calmh** [comment]
At minimum this PR needs some screenshots.

**Comment by wiredmind** [comment]
If not Calibri, Segoe UI is close in size to Arial, and has a "normal" tilde.
Having a themes system is great and all, but why must we have inferior defaults?

**Comment by uok** [comment]
@wiredmind of course you can customize Bootstrap but your PR only consists of one line. Since Calibri displays ~2px smaller, so you need to adjust all (!) font sizes accordingly including button texts - but only for Windows users that have Calibri.
If you don't like the default font, you can use something like [stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/) to adjust every detail

**Comment by wiredmind** [comment]
@uok It may be because I still have good vision, but I don't see a need to adjust the font size. Below is a screenshot of Calibri in use. It's perfectly legible. 

screen shot 2015-11-23 at 14 09 55

Compare with Arial

screen shot 2015-11-23 at 14 11 01

**Comment by calmh** [comment]
It's certainly smaller in x-height than the Arial. although I agree that Calibri looks better otherwise in this screenshot.

both

Arial seems to the be the IE default sans-serif font still, so I'm sure Windows users are used to Arial.

**Comment by wiredmind** [comment]
Pardon the expressive language, but...sure, people are used to Arial.
People were are also used to excrement on the streets, burning other people
alive, and punching holes in paper to make computer programs. Status quo
isn't state of the art and we should strive to improve.

**Comment by calmh** [comment]
Right... At this point I see this as too arbitrary and Windows specific. For example, why Calibri? It's not the Windows system font, it's not the IE default sans-serif font, as far as I can tell it's just used by default for headers in Word? Why not Segoe UI for example?

Not saying Arial is the end all be all of fonts, but perhaps the correct solution here is to either bundle a font, or make some effort to find a font list that matches the expected defaults per platform (Segoe UI, Droid Sans, San Francisco, ...).

**Pull request closed without merging**